superbench/benchmarks/micro_benchmarks/dist_inference.py

```diff
@@ -401,38 +445,79 @@ def _benchmark(self):
         Return:
             True if _benchmark succeeds.
         """
-        batch_size = self._args.batch_size
-        input_size = self._args.input_size
-        hidden_size = self._args.hidden_size
-        num_layers = self._args.num_layers
-        computation = self._args.computation_kernel
-        communication = self._args.communication_kernel
-        activation = self._args.activation_kernel
-        precision = self._args.precision
-        num_warmup = self._args.num_warmup
-        num_steps = self._args.num_steps
-
-        if self.__local_rank == 0:
-            logger.info(
-                'Distributed Inference - using {} GPUs: '
-                'batch_size={}, input_size={}, hidden_size={}, num_layers={}, '
-                'computation_kernel={}, communication_kernel={}, activation_kernel={}, precision={}, '
-                'num_warmup={} num_steps={}'.format(
-                    self.__world_size, batch_size, input_size, hidden_size, num_layers, computation, communication,
-                    activation, precision, num_warmup, num_steps
+        if self._args.use_cpp_impl:
+            # Execute commands if cpp impl path
+            if not super()._benchmark():
+                return False
+            return True
```

Comment:
Why pytorch doesn't need it?

Reply:
This is for compatibility. In order to add support to cpp, the class of this benchmark is changed to MicroBenchWithInvoke, which has its own _benchmark implementation. For PyTorch, we still fall back to our previous own _benchmark implementation.